cmd/activity-monitor/main.go

```diff
@@ -115,7 +115,8 @@ func main() {
 	app.Action = func(c cmd.Config, stats statsd.Statter, auditlogger *blog.AuditLogger) {
 		go cmd.DebugServer(c.ActivityMonitor.DebugAddr)
 
-		ch, err := rpc.AmqpChannel(c)
+		amqpConf := c.ActivityMonitor.AMQP
+		ch, err := rpc.AmqpChannel(amqpConf)
```

Comment:
Seems like it would be cleaner and not that verbose to just make the call directly.

```
ch, err := rpc.AmqpChannel(c.Activitymonitori.AMQP)
```

And likewise below.